src/lib/Microsoft.Health.Events/Repository/StorageManager.cs

```diff
@@ -13,12 +14,10 @@ public class StorageManager : IRepositoryManager
     {
         private BlobContainerClient _blobContainer;
 
-        public StorageManager(string connectionString, string blobContainerName)
+        public StorageManager(Uri containerUri, BlobContainerClient containerClient)
         {
-            EnsureArg.IsNotNull(connectionString);
-            EnsureArg.IsNotNull(blobContainerName);
-
-            _blobContainer = new BlobContainerClient(connectionString, blobContainerName);
+            EnsureArg.IsNotNull(containerUri);
```

Comment:
>containerUri [](start = 32, length = 12)

We have an IsNotNull check but the URI is not used.  Do we still need?

Reply:
Nope. I was leftover from before we passed the client in as an argument - removed the argument and the null check.